app/assets/stylesheets/style/header.scss

```diff
@@ -0,0 +1,203 @@
+.header {
+  border-bottom: solid 1px $light-gray-text;
+  position: relative;
+  z-index: 999;
+
+  .navbar {
+    margin: 0;
+    min-height: inherit;
+
+    .nav {
+      > li > a i {
+        .fa.fa-angle-down {
+          font-size: 12px;
+          left: 1px;
+          margin: 0;
+          position: relative;
+          top: -1px;
+        }
+      }
+
+      > li {
+        .open > a i {
+          .fa.fa-angle-down {
```

Comment:
Nesting should be no greater than 5, but was 6<br>Selector should have depth of applicability no greater than 5, but was 8